### DISCUSSION TITLE
Support for Adol Book 14 with fragerence dispenser,  aka Vivobook M5406UA

### DISCUSSION THREAD

**Discussion by hinchyman** [seerge/g-helper#3085] [Ideas]
First off, ghelper is amazing. I've donated to it but I really feel it's lacking fragrance support. Would it be possible to support the [worlds first laptop with a fragrance dispenser?](https://www.tomshardware.com/laptops/asus-has-a-new-laptop-that-comes-with-a-fragrance-dispenser)

But seriously, I own this. Would it be possible to add support for it? It's basically a rebranded[ ASUS Vivobook S 14 OLED (M5406UA)](https://www.asus.com/us/laptops/for-home/vivobook/asus-vivobook-s-14-oled-m5406/helpdesk_download?model2Name=M5406UA).  I've attached copy of log files if they're useful. It would be amazing if power limits worked.

[log - Copy.txt](https://github.com/user-attachments/files/16938063/log.-.Copy.txt)

**Comment by seerge** [comment]
@hinchyman hello, that’s nice, thanks. But what exactly do you need to be supported here? 

From the log i see that your device doesn’t have any standard controls (like modes) that Asus devices usually have.

Can you change any settings via native apps, like MyAsus?

--

P.S. It looks like you are missing Asus System Control Interface. [Try to reinstall it](https://github.com/seerge/g-helper/wiki/Troubleshooting#reinstalling-asus-system-control-interface)
